Makefile

```diff
@@ -8,5 +10,30 @@ fmt:
 vet:
 	go vet ./...
 
+build-image:
+	${CONTAINER_RUNTIME} build -t ${IMG} .
+
 cmd: fmt vet
 	go build -ldflags="-w -s" -o bin/addon github.com/konveyor/tackle2-addon-windup/cmd
+
+.PHONY: test-e2e
+START_MINIKUBE = ./bin/start-minikube.sh
+INSTALL_TACKLE = ./bin/install-tackle.sh
+start-minikube:
+	mkdir -p $(dir $(START_MINIKUBE)) ;\
+	curl -sSLo $(START_MINIKUBE) https://raw.githubusercontent.com/konveyor/tackle2-operator/main/hack/start-minikube.sh ;\
```

Comment:
I don't think you should download it every time. Maybe you could have something like:

```
.PHONY: start-minikube.sh
START_MINIKUBE_SH = $(shell pwd)/bin/start-minikube.sh
start-minikube.sh: ## Download start-minikube.sh locally if necessary, preferring the $(pwd)/bin path over global if both exist.
ifeq (,$(wildcard $(START_MINIKUBE_SH)))
	@{ \
	set -e ;\
	mkdir -p $(dir $(START_MINIKUBE_SH)) ;\
	curl -sSLo $(START_MINIKUBE_SH) https://raw.githubusercontent.com/konveyor/tackle2-operator/main/hack/start-minikube.sh ;\
	chmod +x $(START_MINIKUBE_SH) ;\
	}
endif
```

then you could have the start-minikube recipe "depend" on start-minikube.sh like:

```
start-minikube: start-minikube.sh
  $(START_MINIKUBE_SH)
```